packages/slider/src/Slider.ts

```diff
@@ -11,23 +11,22 @@ governing permissions and limitations under the License.
 */
 
 import {
+    classMap,
+    CSSResultArray,
     html,
+    ifDefined,
     property,
-    CSSResultArray,
-    TemplateResult,
     query,
-    styleMap,
-    ifDefined,
     repeat,
-    classMap,
+    styleMap,
+    TemplateResult,
 } from '@spectrum-web-components/base';
-
-import sliderStyles from './slider.css.js';
-import { ObserveSlotText } from '@spectrum-web-components/shared/src/observe-slot-text.js';
-import { StyleInfo } from 'lit-html/directives/style-map';
 import '@spectrum-web-components/field-label/sp-field-label.js';
 import type { NumberField } from '@spectrum-web-components/number-field';
+import { ObserveSlotText } from '@spectrum-web-components/shared/src/observe-slot-text.js';
+import { StyleInfo } from 'lit-html/directives/style-map';
 import { HandleController, HandleValueDictionary } from './HandleController.js';
+import sliderStyles from './slider.css.js';
```

Comment:
If this really is just import reordering, can you remove it from this PR?

We should include adding linting to manage order in our 1.0.0 release checklist and have a single _massive_ PR that does this across the library so we dont' end up with this happening in various one of PRs over time.

Reply:
This is from the automatic vscode setting:

```
"editor.codeActionsOnSave": {
        "source.organizeImports": true
    }
```

...iirc there's one that just removes unused imports; I'll swap to that instead. Otherwise I think all of my PRs will get little changes like this auto-added.